pkg/installer/packages/service/datadog_agent.go

```diff
@@ -83,10 +82,6 @@ func SetupAgentUnits(ctx context.Context) (err error) {
 			return
 		}
 	}
-	// write installinfo before start, or the agent could write it
-	if err = installinfo.WriteInstallInfo("updater_package", "manual_update"); err != nil {
-		return
-	}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏Let's add it back